screens/auth/SignUpScreen.js

```diff
@@ -300,21 +306,22 @@ export default class SignUpScreen extends React.Component {
             <AuthTextField
               fieldType="Name"
               value={this.state.values[signUpFields.NAME]}
-              onBlurCallback={(value) =>
-                this.updateError(value, signUpFields.NAME)
-              }
-              changeTextCallback={async (text) =>
-                this.onTextChange(text, signUpFields.NAME)
-              }
+              onBlurCallback={(value) => {
+                this.updateError(value, signUpFields.NAME);
+                this.scrollView.scrollToEnd({ animated: true });
+              }}
+              changeTextCallback={async (text) => {
+                this.onTextChange(text, signUpFields.NAME);
+              }}
               error={this.state.errors[signUpFields.NAME]}
             />
 
             <AuthTextField
               fieldType="Phone Number"
               value={this.state.values[signUpFields.PHONENUM]}
-              onBlurCallback={(value) =>
-                this.updateError(value, signUpFields.PHONENUM)
-              }
+              onBlurCallback={(value) => {
+                this.updateError(value, signUpFields.PHONENUM);
+              }}
```

Comment:
If for some reason the user scrolls up while typing the phone number, perhaps to change their name (such that the sign up button is hidden but the password field is not hidden), tapping on the password field does not animate to show the sign up button – maybe add the blur callback to the phone number field too? Definitely an edge case though

Reply:
Yes nice catch! I added it to both fields now.